### PR TITLE
Update service.markdown

### PR DESCRIPTION
**Pull request by luca-angemi**
**Description:**

notation changed to yaml

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/mqtt/service.markdown [hunk 1]
```diff
@@ -20,38 +20,30 @@ The MQTT integration will register the service `mqtt.publish` which allows publi
...
 ```
 
 {% raw %}
-```json
-{
-  "topic": "home-assistant/light/1/state",
-  "payload_template": "{{ states('device_tracker.paulus') }}"
-}
+```yaml
+topic: "home-assistant/light/1/state
```
Comment (frenck) [inline source/_docs/mqtt/service.markdown]:
```suggestion
topic: home-assistant/light/1/state
```

File: source/_docs/mqtt/service.markdown [hunk 2]
```diff
@@ -20,38 +20,30 @@ The MQTT integration will register the service `mqtt.publish` which allows publi
...
 `payload` must be a string. If you want to send JSON then you need to format/escape it properly. Like:
 
-```json
-{
-  "topic": "home-assistant/light/1/state",
-  "payload":"{\"Status\":\"off\", \"Data\":\"something\"}"
-}
+```yaml
+topic: home-assistant/light/1/state
+payload: {\"Status\":\"off\", \"Data\":\"something\"}
```
Comment (frenck) [inline source/_docs/mqtt/service.markdown]:
```suggestion
payload: "{\"Status\":\"off\", \"Data\":\"something\"}"
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR, @luca-angemi!

Left a couple of comments, could you take a look? Thanks! 👍

**Comment by luca-angemi** [comment]
Changes addressed, sorry about that @frenck!

**Review by frenck — APPROVED** [review]
Looks good now @luca-angemi! 👍 

Happy Hacktoberfest 🎉

**Pull request merged**